src/testE2E/amazonq/featureDev.test.ts

```diff
@@ -0,0 +1,139 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import assert from 'assert'
+import { qTestingFramework } from './framework/framework'
+import { FollowUpTypes } from '../../amazonqFeatureDev/types'
+import sinon from 'sinon'
+import { FeatureDevClient } from '../../amazonqFeatureDev/client/featureDev'
+import { verifyTextOrder } from './framework/text'
+import { examples } from '../../amazonqFeatureDev/userFacingText'
+import * as authUtil from '../../codewhisperer/util/authUtil'
+import request from '../../common/request'
+
+describe('Amazon Q Feature Dev', function () {
+    let framework: qTestingFramework
+
+    const samplePlanResponse = 'sample plan response'
+
+    beforeEach(() => {
+        /**
+         * TODO remove these stubs when we know the backend can handle all the test load + when we know the tests
+         * are working without any flakiness
+         */
+        sinon.stub(authUtil, 'getChatAuthState').resolves({
```

Comment:
I'd recommend using `sinon.createSandbox()`, then stubbing with that sandbox, and using sandbox.restore() in the after. Since you are stubbing static instances it can cross in to other tests and cause issues in the future.